src/allegro_flare/json.cpp

```diff
@@ -405,7 +405,7 @@ namespace allegro_flare
             else if (json[i] == '"')
             {
                std::string s;
-               for (int j = i + 1; j < (int)json.length(); ++j)
+               for (unsigned j = i + 1; j < (int)json.length(); ++j)
```

Comment:
I should note that it's very likely this JSON class will be removed.

I've been using [nlohmann/json](https://github.com/nlohmann/json) in my other projects, and there is a lot of stiff competition right now for competing C++ json libraries.  [This repo does an incredible job](https://github.com/miloyip/nativejson-benchmark) of scoring the different libraries that are out there so far.

I think eventually, if anything AllegroFlare could embed one of those libraries, or wrap one of the libraries in a new interface. nlohmann/json is header-only, which is good for convenience but bad for compile-time speed.  It also spits out warnings which is a bit annoying.

Reply:
ok